data/tutorials/ds_05_seq.md

```diff
@@ -0,0 +1,316 @@
+---
+id: sequences
+title: Sequences
+description: >
+  Learn about an OCaml's most-used, built-in data types
+category: "data-structures"
+date: 2023-01-12T09:00:00-01:00
+---
+
+# Sequences
+
+## Prerequisites
+
+| Concept | Status | Documentation | Reference |
+|---|---|---|---|
+| Basic types | Mandatory | | |
+| Functions | Mandatory | | |
+| Lists   | Mandatory |   |   |
+| Options | Recommended |   |   |
+| Arrays  | Nice to have |   |   |
```

Comment:
I think we need to upgrade omd to be able to use tables or so? :thinking:

Maybe it's better for now to summarize:

You should be comfortable with writing functions over [lists](https://ocaml.org/docs/lists) and, ideally, understand what an [option](https://ocaml.org/docs/data-types#example-options) is.